src/fswAlgorithms/effectorInterfaces/rwMotorTorque/rwMotorTorque.c

```diff
@@ -91,6 +91,7 @@ void Update_rwMotorTorque(rwMotorTorqueConfig *configData, uint64_t callTime, in
 {
```

Comment:
:small_red_triangle_down: There is an unused `<string.h>` include in this file. I suggest we add a commit to the end of this branch which removes it.

Reply:
Done